engine/storage/integration-test/test/org/apache/cloudstack/storage/test/FakePrimaryDataStoreDriver.java

```diff
@@ -44,7 +45,7 @@
     boolean snapshotResult = true;
 
     @Override
-    public ChapInfo getChapInfo(VolumeInfo volumeInfo) {
+    public ChapInfo getChapInfo(DataObject dataObject) {
```

Comment:
This appears to be the default implementation from the IDE.  I realize that it predates this patch, but should method do more?  If not, can we strike the IDE generated comment to avoid future confusion?